config/application.rb

```diff
@@ -29,6 +29,10 @@ class Application < Rails::Application
     config.plugins = [:dynamic_form]
 
     config.eager_load_paths << Rails.root.join('lib')
+
+    initializer(:remove_activestorage_routes, after: :add_routing_paths) do |app|
```

Comment:
Isn't there easier way to disable `activestorage`? What about just not loading it?

Reply:
we will have to stop using `require 'rails/all'` and do it something like [this](https://stackoverflow.com/questions/49813214/disable-active-storage-in-rails-5-2#answer-49826159)

Reply:
Maybe load only needed parts instead of `rails/all` in `applicaiton.rb` ?

https://github.com/rails/rails/blob/master/railties/lib/rails/all.rb
https://github.com/rubygems/rubygems.org/blob/master/config/application.rb#L3

Reply:
Using initializer seemed cleaner than the alternative. Should I really change it to require lines?

Reply:
It is standard way in my :eyes:  to replace `rails/all` with requires you only need when you don't need them all (and it is the same `--skip-*` flags do during `rails new`).

Reply:
updated! thanks.